src/main/java/gyro/google/compute/ComputeTags.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.api.services.compute.model.Tags;
+import gyro.core.resource.Diffable;
+import gyro.google.Copyable;
+
+public class ComputeTags extends Diffable implements Copyable<Tags> {
+
+    private String fingerprint;
+
+    private List<String> items;
+
+    /**
+     * Specifies a fingerprint for this request, which is essentially a hash of the tags' contents and
+     * used for optimistic locking.
+     * The fingerprint is initially generated by Compute Engine and changes after every request to modify or update tags.
+     * You must always provide an up-to-date fingerprint hash in order to update or change tags.
+     */
+    public String getFingerprint() {
+        return fingerprint;
+    }
+
+    public void setFingerprint(String fingerprint) {
+        this.fingerprint = fingerprint;
+    }
+
+    /**
+     * Tag items. Each tag must be 1-63 characters long, and comply with RFC1035.
+     */
+    public List<String> getItems() {
```

Comment:
Maybe have the `@Regex` validator on here. The validation for collection happens for each item.